src/Codebreaker.WPF/Views/Components/GameResultDisplay.xaml.cs

```diff
@@ -0,0 +1,74 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Windows.Input;
+
+namespace Codebreaker.WPF.Views.Components;
+
+/// <summary>
+/// Interaction logic for GameResultDisplay.xaml
+/// </summary>
+[ObservableObject]
+public partial class GameResultDisplay : UserControl
+{
+    public GameResultDisplay()
+    {
+        InitializeComponent();
+    }
+
+    public GameMode GameMode
+    {
+        get => (GameMode)GetValue(GameModeProperty);
+        set
+        {
+            SetValue(GameModeProperty, value);
+            Debug.WriteLine(value);
+            OnPropertyChanged(nameof(GameMode));
+        }
+    }
+
+    public static readonly DependencyProperty GameModeProperty =
+        DependencyProperty.Register(nameof(GameMode), typeof(GameMode), typeof(GameResultDisplay), new PropertyMetadata(GameMode.NotRunning, OnGameModeChanged));
+
+    public ICommand? LostCommand
+    {
+        get => (ICommand?)GetValue(LostCommandProperty);
+        set => SetValue(LostCommandProperty, value);
+    }
+
+    public static readonly DependencyProperty LostCommandProperty =
+        DependencyProperty.Register(nameof(LostCommand), typeof(ICommand), typeof(GameResultDisplay), new (OnLostCommandChanged));
+
+    public Visibility LostButtonVisibility => LostCommand is not null ? Visibility.Visible : Visibility.Collapsed;
```

Comment:
Instead of setting the visibility of every button here, we should consider using VSM (I think an issue is already active)

Reply:
I will solve this in #99